validator/src/cli.rs

```diff
@@ -212,6 +212,41 @@ fn deprecated_arguments() -> Vec<DeprecatedArg> {
             .long("enable-quic-servers"),
         usage_warning: "The quic server is now enabled by default.",
     );
+    // All etcd config is deprecated as of v2.2
+    add_arg!(Arg::with_name("etcd_endpoint")
+        .long("etcd-endpoint")
+        .required_if("tower_storage", "etcd")
+        .value_name("HOST:PORT")
+        .takes_value(true)
+        .multiple(true)
+        .validator(solana_net_utils::is_host_port)
+        .help("etcd gRPC endpoint to connect with"),);
+    add_arg!(Arg::with_name("etcd_domain_name")
+        .long("etcd-domain-name")
+        .required_if("tower_storage", "etcd")
+        .value_name("DOMAIN")
+        .default_value("localhost")
```

Comment:
This value copied from below spot to avoid tunneling `DefaultArgs` in. This seemed reasonable given that the item is deprecated:
https://github.com/anza-xyz/agave/blob/e9e0e40958c87275613a32c26cc52db9d6608fd6/validator/src/cli.rs#L435